Gemfile.lock

```diff
@@ -1,26 +1,26 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    docile (1.3.2)
+    coderay (1.1.3)
```

Comment:
What does this gem do?

Reply:
I have no clue... It might be included with the Sinatra gem.

Reply:
ooo interesting.